source/MaterialXView/Viewer.cpp

```diff
@@ -124,6 +124,52 @@ void applyModifiers(mx::DocumentPtr doc, const DocumentModifiers& modifiers)
     }
 }
 
+// ViewDir implementation for GLSL
+// as needed for the environment shader.
+class ViewDirGlsl : public mx::GlslImplementation
```

Comment:
Based on the old viewdirection implementation. Now moved here so it's only available for the viewer.